src/main/webapp/find-events.html

```diff
@@ -29,13 +29,14 @@
       </div>
     </header>
 
-    <h1 class="title">find events</h1>
-
     <!-- main content -->
-    <div id="page-container">
-      <div id="map"></div>
-      <div id="all-posts">
-        <button id="create-post-button" type="button">create a post</button>
+    <div id="col-container">
+	    <h1>find events</h1>
```

Comment:
Check indent here.

Reply:
It seems fine on my end but not sure why on GitHub, it indented weirdly!